src/MoBi.Presentation/Views/IPathAndValueEntitiesView.cs

```diff
@@ -25,16 +25,15 @@ public interface IPathAndValueEntitiesView<TStartValueDTO> : IPathAndValueEntiti
       void AddPathItems(IEnumerable<string> pathValues);
 
       void ClearPathItems();
-      IReadOnlyList<TStartValueDTO> SelectedStartValues { get; }
-      IReadOnlyList<TStartValueDTO> VisibleStartValues { get; }
-      void AddDeleteStartValuesView(IView view);
-      void HideDeleteView();
+
+      void HideDeleteButton();
```

Comment:
this all feels a bit verbose here @rwmcintosh Like 6 different variables to hide (and not show)
Woild it make sense to havean Hide (Delete | Presence | Button|Refreh) etc.